classes/class-ppc-pagesetups.php

```diff
@@ -170,23 +333,25 @@ public function ppc_custom_box_html() {
 					</div>
 				</div>
 				<?php
-				foreach ( $ppc_checklist_item_data as $ppc_key => $ppc_value ) {
+				foreach ( $ppc_checklist_item_data[ $ppc_screen ] as $ppc_key => $ppc_value ) {
 					?>
 					<label for="<?php echo esc_attr( $ppc_key ); ?>">
-					<div class="ppc-checklist-item-wrapper">
-						<input type="checkbox" name="checkbox[]" id="<?php echo esc_attr( $ppc_key ); ?>" class="ppc_checkboxes" value= "<?php echo esc_attr( $ppc_key ); ?>" <?php empty( $value ) ? '' : checked( true, array_key_exists( $ppc_key, $value ) ); ?>
-						>
-					<div class="ppc-checklist"><?php echo esc_attr( $ppc_value ); ?></div></div>
-				</label>
-					<?php
+						<div class="ppc-checklist-item-wrapper">
+							<input type="checkbox" name="checkbox[]" id="<?php echo esc_attr( $ppc_key ); ?>" class="ppc_checkboxes" value= "<?php echo esc_attr( $ppc_key ); ?>" 
+							<?php
+							empty( $value ) ? '' : checked( true, array_key_exists( $ppc_key, $value ) );
```

Comment:
1. Rename the variable `$value` to be something relevant to the value it holds.
2. Check if ternary condition here can be avoided. The function `checked` does return empty string.

Reply:
Variable renamed.

Reply:
If I changed ternary operator `**empty( $value ) ? '' : checked( true, array_key_exists( $ppc_key, $value ) );**`  to **checked( true, array_key_exists( $ppc_key, $value ) );** it throws an error if the user does not check any item in post metabox.

Reply:
What error you are facing here?

Reply:
https://share.getcloudapp.com/rRuLkgX6

Reply:
Retained:
empty( $value ) ? '' : checked( true, array_key_exists( $ppc_key, $value ) );